.github/workflows/lint-scripts/websites-shared-credentials-sort-order.rb

```diff
@@ -0,0 +1,14 @@
+require 'json'
+
+shared_websites = JSON.parse(File.read('quirks/websites-with-shared-credential-backends.json'))
+
+shared_websites_sorted = shared_websites.sort_by(&:first)
+
+File.open('quirks/websites-with-shared-credential-backends.json','w') do |f|
+  f.write(JSON.pretty_generate(shared_websites_sorted))
```

Comment:
Can we maintain an indentation level of four spaces?

Reply:
Ruby defaults to 2 spaces - https://stackoverflow.com/questions/6741586/rails-coding-standards-why-2-space-indentation

Do you still want 4?

Reply:
I’m sorry! I meant for the data itself, not your action’s code. I think `JSON.pretty_generate` can take an indentation level.

Reply:
Added 4-space indent to JSON formatter.